tests/integration/targets/lvm_pv_move_data/tasks/creation.yml

```
@@ -0,0 +1,99 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: Creating a 500MB file for the first loop device
+  ansible.builtin.command: dd if=/dev/zero of={{ remote_tmp_dir }}/test_lvm_pv_01.img bs=1M count=500
+  args:
+    creates: "{{ remote_tmp_dir }}/test_lvm_pv_01.img"
+
+- name: Creating a 1000MB file for the second loop device
+  ansible.builtin.command: dd if=/dev/zero of={{ remote_tmp_dir }}/test_lvm_pv_02.img bs=1M count=1000
+  args:
+    creates: "{{ remote_tmp_dir }}/test_lvm_pv_02.img"
+
+- name: Creating loop device
+  ansible.builtin.command: losetup -f
+  register: loop_device_01
+
+- name: Associating loop device with file
+  ansible.builtin.command: 'losetup {{ loop_device_01.stdout }} {{ remote_tmp_dir }}/test_lvm_pv_01.img'
+
+- name: Creating loop device
+  ansible.builtin.command: losetup -f
+  register: loop_device_02
+
+- name: Associating loop device with file
+  ansible.builtin.command: 'losetup {{ loop_device_02.stdout }} {{ remote_tmp_dir }}/test_lvm_pv_02.img'
+
+- name: Wiping existing LVM metadata
+  community.general.lvm_pv:
+    device: "{{ item }}"
+    force: true
+    state: absent
+  loop:
+    - "{{ loop_device_01.stdout }}"
+    - "{{ loop_device_02.stdout }}"
+
+- name: Creating physical volume for the first loop device
+  community.general.lvm_pv:
+    device: "{{ loop_device_01.stdout }}"
+  register: pv_creation_result_01
+
+- name: Checking the first physical volume size
+  ansible.builtin.command: pvs --noheadings -o pv_size --units M {{ loop_device_01.stdout }}
+  register: pv_size_output_01
+
+- name: Creating physical volume for the second loop device
+  community.general.lvm_pv:
+    device: "{{ loop_device_02.stdout }}"
+  register: pv_creation_result_02
+
+- name: Checking the second physical volume size
+  ansible.builtin.command: pvs --noheadings -o pv_size --units M {{ loop_device_02.stdout }}
+  register: pv_size_output_02
+
+- name: Creating volume group on top of first device {{ loop_device_01.stdout }}
+  community.general.lvg:
+    vg: testvg
+    pvs: "{{ loop_device_01.stdout }}"
+    force: true
+  register: vg_creation_result
+
+- name: Creating LV testlv on VG testvg
+  community.general.lvol:
+    vg: testvg
+    lv: testlv
+    size: 100%FREE
+    force: true
+  register: lv_creation_result
+
+- name: Creating xfs filesystem on LV testlv
+  community.general.filesystem:
+    dev: /dev/testvg/testlv
+    fstype: xfs
+    state: present
+  register: fs_creation_result
+
+- name: Mounting LV testlv
+  ansible.posix.mount:
+    fstab: '{{ remote_tmp_dir }}/fstab'
+    path: '{{ remote_tmp_dir }}/tmp_mount'
+    src: '/dev/testvg/testlv'
+    fstype: xfs
+    opts: rw,noauto
+    state: mounted
+
+- name: Asserting physical volume was created
+  ansible.builtin.assert:
+    that:
+      - pv_creation_result_01.changed == true
+      - pv_creation_result_02.changed == true
+      - vg_creation_result.changed == true
```

Use the `changed` test:
```suggestion
      - pv_creation_result_01 is changed
      - pv_creation_result_02 is changed
      - vg_creation_result is changed
```